doc/xml/reference.xml

```diff
@@ -363,6 +363,48 @@
                         <example>sas</example>
                     </config-key>
 
+                    <!-- ======================================================================================================= -->
+                    <config-key id="repo-gcs-bucket" name="GCS Repository Bucket">
+                        <summary>GCS repository container.</summary>
```

Comment:
We have `repo-s3-bucket` and the summary and description mention `bucket`. We have `repo-azure-container` and the summary and description mention `container`. So what is gcs - a bucket or container?

Reply:
Should be bucket. Fixed in a3bdc970a5fc60b4bced491f.